src/util/community-addons.js

```diff
@@ -0,0 +1,42 @@
+const communityAddons = [
```

Comment:
You could simplify this to:

```
const communityAddons = {
  'design-assets': 'https://github.com/storybookjs/addon-design-assets',
  'events': 'https://github.com/storybookjs/addon-events',
  // ...
};
```

That way you don't have to do a find for every addon.

Reply:
@winkerVSbecks just pushed a commit to address the feedback. Care to take another look?